language-support/java/codegen/src/it/java-1.3/com.digitalasset.lf1_3/AllTests.java

```diff
@@ -10,6 +10,7 @@
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
         MapTest.class,
+        OptionalTest.class,
```

Comment:
@gerolf-da Instead of moving the test to the DAML-LF 1.3 test suite, we could also check in a `.dalf` with DAML-LF 1.1 in it.